tvb_contrib/tvb/contrib/scripts/utils/data_structures_utils.py

```diff
@@ -868,24 +871,28 @@ def data_xarray_from_continuous_events(events, times, senders, variables=[],
         from xarray import DataArray
```

Comment:
Return types depending on the installed set of packages is a difficult debug session waiting to happen, unless the two (dict and DataArray) behave the same, in which case, only the dict is required.  Please move this out to the call site